docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,151 @@
+# Deploying a production network
+
+This deployment guide is a high level overview at the proper sequence for setting up a functioning Fabric network, in addition to best practices and a few of the many considerations to keep in mind when deploying. Note that this topic will discuss "setting up the network" as a holistic process from the perspective of a single individual. More likely than not, real world production networks will not be set up by a single individual but as a collaborative effort directed by several individuals (a collection of banks each setting up their own components, for example) instead.
+
+The process for deploying a Fabric network is difficult and presumes a high degree of expertise in database management and cryptographic methods (especially x.509 certificates). If you are a smart contract or application developer, you should not need this level of expertise in deploying a production level Fabric network. However, you might need to be aware how networks are deployed in ordering to develop effective smart contracts and applications.
```

Comment:
Not sure they need to know about database management. Understanding of network and (most likely) containers is more important. I would say:
presumes and understanding of Public Key Infrastructure and managing distributed systems.